extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -29,10 +29,7 @@ class UpdateReplicationStatus extends BackbeatTask {
      */
     constructor(rsp, metricsHandler) {
         const rspState = rsp.getStateVars();
-        super({
-            timeoutS:
-            rspState.repConfig.replicationStatusProcessor.retryTimeoutS,
-        });
+        super(rspState.repConfig.replicationStatusProcessor.retry);
```

Comment:
retryParams are already on line 34, what is the point of this change?

(a cleanup may be good indeed, to pass the retryParams through the constructor instead of overriding them: but this is not what you are implementing here, and would change most backbeat tasks)


```suggestion
        super();
```

Reply:
Yes, that was my point about passing it in the constructor. I overlooked the unnecessary redundancy and will remove `this.retryParams = this.repConfig.replicationStatusProcessor.retry;`

Thanks.